app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        vaccine = visitor.get("vaccine")
+        if not vaccine:
+            raise NotVaccinatedError("Visitor does't have vaccine.")
+        if date.today() > vaccine["expiration_date"]:
```

Comment:
use get() here also

Reply:
If I get None, I still get an error

Reply:
Do you can explain me, why use get here

Reply:
Here is more about style of code: if you use get() in one place, use it in all places. And you will not have KeyError